<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="hp" failures="1" tests="2" name="null" time="0.787" errors="0" timestamp="2023-03-04T00:44:59 IST">
  <testcase classname="com.bridgelabz.TestFailure" name="testPassed" time="0.003"/>
  <testcase classname="com.bridgelabz.TestFailure" name="testFailed" time="0.007">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.bridgelabz.TestFailure.testFailed(TestFailure.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- testFailed -->
  <testcase classname="com.bridgelabz.TestFailure" name="testDepend" time="0.0">
    <ignored/>
  </testcase> <!-- testDepend -->
</testsuite> <!-- null -->
